### PR TITLE
Rename GraphQLSchema CR to GraphQLApi

### PR DESCRIPTION
**Pull request by jenshu**
# Description

Rename GraphQLSchema to GraphQLApi, which is a more appropriate description of the custom resource.

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [ ] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: changelog/v1.11.0-beta21/graphql-rename.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+changelog:
+  - type: NON_USER_FACING
```
Comment (jenshu) [inline changelog/v1.11.0-beta21/graphql-rename.yaml:2]: not sure of the type - non user facing? breaking?

**Review by kdorosh — COMMENTED** [review]
seems reasonable to me!

**Comment by kdorosh** [comment]
/kick ssl e2e test flake https://storage.googleapis.com/solo-public-build-logs/logs.html?buildid=89162e79-f7b5-40d5-88e7-cfaf3b4ad3ae

perhaps @jenshu you can look into this flake while ci is running? I've seen a bunch of other PRs hit this as well

**Review by kdorosh — APPROVED** [review]

File: install/helm/gloo/crds/graphql.gloo.solo.io_v1alpha1_GraphQLApi.yaml [hunk 2]
```diff
@@ -4,17 +4,17 @@ apiVersion: apiextensions.k8s.io/v1
...
-    plural: graphqlschemas
+    kind: GraphQLApi
+    listKind: GraphQLApiList
+    plural: graphqlapis
     shortNames:
-    - gqls
-    - ggqls
-    singular: graphqlschema
+    - gqla
+    - ggqla
```
Comment (saiskee) [inline install/helm/gloo/crds/graphql.gloo.solo.io_v1alpha1_GraphQLApi.yaml:16]:
```suggestion
    - gql
    - gqls
```

**Review by saiskee — CHANGES_REQUESTED** [review]

File: install/helm/gloo/crds/graphql.gloo.solo.io_v1alpha1_GraphQLApi.yaml (hunk 2 above)
Comment (jenshu) [reply install/helm/gloo/crds/graphql.gloo.solo.io_v1alpha1_GraphQLApi.yaml]: (spoke offline - changed to gql / ggql)

**Pull request merged**